proposals/description/fip_79.ts

```diff
@@ -0,0 +1,35 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_79: ProposalDescription = {
+  title: 'FIP-79: Tribe x Olympus Partnership',
+  commands: [
+    {
+      target: 'core',
+      values: '0',
+      method: 'allocateTribe(address,uint256)',
+      arguments: ['{feiDAOTimelock}', '3747957360000000000000000'],
+      description: 'Send 3.74M TRIBE to Fei DAO Timelock'
+    },
+    {
+      target: 'tribe',
+      values: '0',
+      method: 'approve(address,uint256)',
+      arguments: ['{ohmEscrow}', '3747957360000000000000000'],
```

Comment:
I'm not seeing `ohmEscrow` anywhere in mainnetAddresses.ts

Reply:
its deployed and exported in the FIP-79 script. Will be added once the initial review is complete